README.md

```diff
@@ -13,3 +13,97 @@ Please see the [contributing guide](https://github.com/exercism/docs/tree/master
+
+## Exercise Tests ##
+
+At the most basic level, Exercism is all about the tests. They drive the user's implementation forward and tell them when the exercise is complete.
+
+The utmost care and attention should be used when adding or making changes to the tests for an exercise. When implementing an exercise test suite, we want to provide a good user experience for the people writing a solution to the exercise. People should not be confused or overwhelmed.
+
+We simulate Test-Driven Development (TDD) by implementing the tests in order of increasing complexity. We try to ensure that each test either
+
+- helps triangulate a solution to be more generic, or
+- requires new functionality incrementally.
+
+Test files should use the following format:
+
+```
+extern crate exercise_name;
+
+use exercise_name::*;
+
+#[test]
+fn test_descriptive_name() {
+    assert_eq!(exercise_function(1), 1);
+}
+
+#[test]
+#[ignore]
+fn test_second_and_past_tests_ignored() {
+    assert_ne!(exercise_function(1), 2);
+}
+```
+
+## Submitting a Pull Request ##
+
+Please keep the following in mind:
+
+- Pull requests should be focused on a single exercise, issue, or change.
+
+- We welcome changes to code style, and wording. Please open a separate PR for these changes if possible.
+
+- Please open an issue before creating a PR that makes significant (breaking) changes to an existing exercise or makes changes across many exercises. It is best to discuss these changes before doing the work. Discussions related to exercises that are not track specific can be found in [exercism/discussions](https://github.com/exercism/discussions/issues).
+
+- Follow the coding standards for Rust.  [rustfmt](https://github.com/nrc/rustfmt) may help with this
+and can be installed with `cargo install rustfmt`.
+
+- Watch out for trailing spaces, extra blank lines, and spaces in blank lines.
+
+- All the tests for Rust exercises can be run from the top level of the repo with `_test\check-exercises.sh` Please run this command before submitting your PR.  See [Windows Specific instructions](_test/WINDOWS_README.md) for running this.
+
+## Contributing a New Exercise ##
+
+- All Exercism exercises must be defined in [problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises) before they are implemented for a specific track. Please submit a PR there if your exercise is new to Exercism.
+
+- Please make sure the new exercise conforms to specifications in the [exercism/problem-specifications](https://github.com/exercism/problem-specifications) repo.  (Note if slight changes are needed for Rust
```

Comment:
I would say `/` instead of `\` in `.meta\hints.md`, for consistency with other instances of directory separator in this file (`exercism/discussions`, `exercises/exercise/name`, etc.)